src/main/org/apache/tools/ant/taskdefs/optional/junitlauncher/StandaloneLauncher.java

```diff
@@ -204,6 +215,8 @@ public Properties getProperties() {
         private boolean haltOnFailure;
         private List<TestDefinition> tests = new ArrayList<>();
         private List<ListenerDefinition> listeners = new ArrayList<>();
+		private List<String> includesList = new ArrayList<>();
```

Comment:
Minor - can you name this `includeTags`. Same with the other one - `excludeTags`